.travis.yml

```diff
@@ -31,10 +30,15 @@ env:
   - TOXENV=py33-django16
   - TOXENV=py33-django17
   - TOXENV=py33-django18
+  - TOXENV=py34-django110
+  - TOXENV=py34-django111
   - TOXENV=py34-django17
   - TOXENV=py34-django18
   - TOXENV=py34-django19
-  - TOXENV=py34-django110
+  - TOXENV=py35-django110
+  - TOXENV=py35-django111
   - TOXENV=py35-django18
   - TOXENV=py35-django19
-  - TOXENV=py35-django110
+  - TOXENV=py36-django111
+  - TOXENV=pylint-py27-django111
+  - TOXENV=pylint-py35-django111
```

Comment:
I know what the script does is one thing, but ... can you make a true version sort, i.e. Django 1.8 < 1.9 < 1.10 < 1.11 grouped by Python 2.7 < 3.3 < 3.4 < 3.5 < 3.6?
```
...
- TOXENV=py34-django17
- TOXENV=py34-django18
- TOXENV=py34-django19
- TOXENV=py34-django110  # must be placed here manually
- TOXENV=py34-django111  # must be placed here manually, too
- TOXENV=py35-django18
- TOXENV=py35-django19
- TOXENV=py35-django110  # same as above
- TOXENV=py35-django111  # same as above, again
...
- TOXENV=py36-django1xx  # Django 1.8, 1.9, 1.10 should have been added by the script
...
- TOXENV=py36-django111
```
Thanks for your understanding!